app/code/Magento/Catalog/etc/frontend/di.xml

```diff
@@ -82,20 +82,11 @@
     <type name="Magento\Catalog\Block\Product\View\Gallery">
         <arguments>
             <argument name="galleryImagesConfig" xsi:type="array">
-                <item name="small_image" xsi:type="array">
```

Comment:
Hi
Looks like you accidentally removed working part of code, can you please review your changes? As reference you can use this commit from original PR 
https://github.com/magento/magento2/pull/13862/files#diff-da4fdd9a985964091b2bbe0a3722ab77